weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/DatasetEditorContext.tsx

```diff
@@ -1,24 +1,28 @@
-import isEqual from 'lodash/isEqual';
 import React, {createContext, useCallback, useContext, useState} from 'react';
 
-import {flattenObjectPreservingWeaveTypes} from '../flattenObject';
-
 export interface DatasetRow {
```

Comment:
you know i'm obsessed with micro functions, but it might be easier to understand if there were helper methods to access these `__weave` items, like: 

`addEditedFieldToRow` etc. 

Just a thought ;)